### DISCUSSION TITLE
error in displaying visuals in built in webapp

### DISCUSSION THREAD

**Discussion by Muqtadar08** [vanna-ai/vanna#245] [Q&A]
Error
module 'vanna' has no attribute 'run_sql_is_set'
![image](https://github.com/vanna-ai/vanna/assets/155961206/96667559-63b1-4838-811c-def2de6b8eb9)

**Comment by zainhoda** [comment]
It appears that you’re using an old tutorial. Could you provide the rest of the code and I can show you how to change it
